src/js/rebuses.js

```diff
@@ -174,6 +174,10 @@ const rebuses = [
   {
     symbols: ['sc', '+', '🦍', '+', '🐐'],
     words: ['scapegoat']
+  },
+  {
+    symbols: ['🐜', '+', '⛵️', '+' 'tica'],
```

Comment:
Also: add a `,` after the `'+'` in your array

Reply:
Thanks for your patience with my careless typing!